CHARTER.md

```diff
@@ -0,0 +1,89 @@
+
+The TSC shall meet regularly using tools that enable participation by the community. The meeting shall be directed by the TSC Chairperson. Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+TSC voting members are expected to regularly participate in TSC activities.
+
+A TSC voting member is automatically converted to a TSC regular member if they do not participate in three consecutive TSC votes.
+
+## **Section 4: Roles & Responsibilities**
+
+Subject to such policies as may be set by the CPC, the TSC is responsible for all technical development within the [cosmos.gl](http://cosmos.gl)  project, including:
+
+- Setting release dates.  
+- Release quality standards.  
+- Technical direction.  
+- Project governance and process.  
+- GitHub repository hosting.  
+- Conduct guidelines.  
+- Maintaining the list of additional Collaborators.  
+- Development process and any coding standards.  
+- Mediating technical conflicts between Collaborators or Foundation projects.
+
+The TSC will define cosmos.gl project’s release vehicles.
+
+### **Section 4.1 Project Operations & Management**
+
+The TSC will establish and maintain a development process for the cosmos.gl project. The development process will establish guidelines for how the developers and community will operate. It will, for example, establish appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance of a TSC meeting).
+
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### **Section 4.2: Decision-making, Voting, and/or Elections**
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
+
+The TSC follows a [Consensus Seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) decision making model among voting members. When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, the winning candidate option is the one that wins a simple majority of all TSC voting members in every head-to-head election against each of the other candidates. A TSC voting member may choose to participate in any vote through abstention. As long as a vote is open, TSC voting members' choices must not be disclosed, to avoid influencing other voting members.
+
+## **Section 4.3: Other Project Roles**
```

Comment:
You can simply omit sections that are not needed.

Reply:
Omited